platform/components/spire/values.yaml

```diff
@@ -5,6 +5,9 @@ server:
 agent:
   kubeletSkipVerification: true
 
+csi:
```

Comment:
This will require changes to Tekton Chains to use the CSI driver. I assume it will be a separate PR?

Reply:
There's an update to patch_spire.json in this PR that changes `hostPath` to `csi`.

Reply:
missed that. Thank you!